TestProjects/BatchRendererGroup_URP/ProjectSettings/EditorSettings.asset

```diff
@@ -22,7 +22,7 @@ EditorSettings:
     inProgressEnabled: 1
   m_EnableTextureStreamingInEditMode: 1
   m_EnableTextureStreamingInPlayMode: 1
-  m_AsyncShaderCompilation: 0
+  m_AsyncShaderCompilation: 1
```

Comment:
Are you sure about this one? Async compilation is usually disable in test project as it was causing various issue with uncompiled shader and also with SRP batcher. Any reasons to enable it?

Reply:
Yes, it should be good. This PR is only the Graphics part of the error/loading material implementation. It is dependent on the associated C++ PR which aims to provide good support for BRG error/loading material and also improve SRP Batcher behaviour when dealing with pathological shaders (e.g shader not compiled, shader compilation errors, not SRP Batcher compatible, missing `DOTS_INSTANCING_ON`, etc). Most of the changes are located in the C++ PR.